### ISSUE TITLE
Permission denied to manifest-fix.temp

### ISSUE THREAD

**Issue by Lyubomyr** [capistrano/rails#129]
I had issue with manifest file, so make upgrade to last version.
After that new issue appeared:
/public/assets/manifest-fix.temp: Permission denied

Capistrano can't get access to manifest-fix.temp, which `deploy:fix_absent_manifest_bug` created.
Only after comment out this line
> after 'deploy:updating', 'deploy:fix_absent_manifest_bug'

deploy back to work.

Using:
ruby (2.2.2)
rails (4.2.3)
sprockets (3.2.0)
capistrano (3.4.0)
capistrano-bundler (1.1.4)
capistrano-rails (1.1.3)

**Comment by will-in-wi** [comment]
@Lyubomyr are you still having this issue?

**Comment by Lyubomyr** [comment]
Don't remember :) Close it please.
